### PR TITLE
add(example): add a roger bot runs on wechaty telegram bot adaptor

### PR DESCRIPTION
**Pull request by hczhcz**

### CODE REVIEW

**Comment by zixia** [comment]
Awesome!

Could you please also update the Example Wiki Page at here? https://github.com/Chatie/wechaty/wiki/Example

Thanks!

**Comment by hczhcz** [comment]
@zixia Got it. I have updated the Wiki.

**Comment by zixia** [comment]
Great!

**Comment by suntong** [comment]
@hczhcz, so what does this `telegram-roger-bot.js` do? 

> Roger Bot Uses Wechaty Telegram Bot Adaptor

which I found at https://github.com/hczhcz/wechaty-telegram

however, on looking into the readme, I still not quite understand what this _Wechaty Telegram Bot Adaptor_ does. 

Could you list what it is capable of (and maybe typical application scenario) please? Thx!

**Comment by hczhcz** [comment]
@suntong 

TL;DR:

> Run your Telegram bot on WeChat!

The purpose is to run pre-exist Telegram bots on WeChat. If you have written a Telegram bot which uses `node-telegram-bot-api`, simply using the bot adaptor and you will get a WeChat bot.

Technically, it provides a different API from WeChaty's but runs on the same infrastructure as in WeChaty.

---

To forward messages between WeChat and Telegram, you need two bots (one for WeChat and another for Telegram).

```javascript
const WechatyTelegramBot = require('wechaty-telegram');
const TelegramBot = require('node-telegram-bot-api');

const bot1 = new WechatyTelegramBot(...);
const bot2 = new TelegramBot(...);
```

Once a message comes into a bot, send it via another bot

```javascript
bot1.on('message', (msg) => {
    bot2.sendMessage(...);
});
bot2.on('message', (msg) => {
    bot1.sendMessage(...);
});
```

That is probably what you are looking for.

**Comment by suntong** [comment]
Oh, thanks A LOT @hczhcz !!
